qcodes_contrib_drivers/drivers/Attocube/ANC350.py

```diff
@@ -0,0 +1,510 @@
+            positioner can be varied. Value for the drive voltage of the piezo drive. Bychanging this value, the step
+            size of the positioner can be varied.
+        status:
+        voltage: Sets the DC level on the voltage output when no sawtooth based motion and no feedback loop
+            is active.
+        target_position: Sets the target position for automatic motion (start_auto_move). For linear type actuators the
+            position unit is m, for goniometers and rotators it is degree.
+        target_range: Defines the range around the target position where the target is considered to be reached.
+        actuator: Selects the actuator to be used for the axis from actuator presets.
+        actuator_name: Get the name of the currently selected actuator
+        capacitance: Performs a measurement of the capacitance of the piezo motor and returns the result. If no
+            motor is connected, the result will be 0.
+            The function doesn't return before the measurement is complete; this will take a few seconds of time.
+    """
+
+    def __init__(self, parent: "ANC350", name: str, axis: int):
+        super().__init__(parent, name)
+
+        self._axis = axis
+
+        self.add_parameter("position",
+                           label="Position",
+                           get_cmd=self._get_position,
+                           unit="mm or m°")
+
+        self.add_parameter("frequency",
+                           label="Frequency",
+                           get_cmd=self._get_frequency,
+                           set_cmd=self._set_frequency,
+                           vals=vals.Ints(1, 5000),
+                           unit="Hz")
+
+        self.add_parameter("amplitude",
+                           label="Amplitude",
+                           get_cmd=self._get_amplitude,
+                           set_cmd=self._set_amplitude,
+                           vals=vals.Numbers(0, 70),
+                           unit="V")
+
+        self.add_parameter("status",
+                           label="Status",
+                           get_cmd=self._get_status)
+
+        self.add_parameter("target_position",
+                           label="Target Position",
+                           get_cmd=None,
+                           set_cmd=self._set_target_position,
+                           vals=vals.Numbers(),
+                           unit="mm or m°")
+
+        self.add_parameter("target_range",
+                           label="Target Range",
+                           get_cmd=None,
+                           set_cmd=self._set_target_range,
+                           vals=vals.Numbers(),
+                           unit="mm or m°")
+
+        self.add_parameter("actuator",
+                           label="Actuator",
+                           get_cmd=None,
+                           set_cmd=self._set_actuator,
+                           vals=vals.Numbers(0, 255))
```

Comment:
Are you sure this can be both integer and float? To me, it is most likely to be only integer (vals.Ints(0, 255)), but I couldn't find it explicitly in the manual.

Reply:
Yes, you're right. I will change it.